packages/picker/src/Picker.ts

```diff
@@ -406,6 +412,15 @@ export class PickerBase extends SizedMixin(Focusable) {
                           ></sp-icon-alert>
                       `
                     : nothing}
+                ${this.loading
+                    ? html`
+                          <sp-progress-circle
+                              indeterminate
+                              aria-labelledby="label"
+                              aria-hidden="true"
+                          ></sp-progress-circle>
+                      `
+                    : nothing}
```

Comment:
Just a suggestion, instead of using ternary operators in which we need to inform `nothing` as "else", we could use the `when` directive:
```html
                ${when(
                    this.loading,
                    () => html`
                        <sp-progress-circle
                            indeterminate
                            aria-hidden="true"
                        ></sp-progress-circle>
                    `
                )}
```
😁